experimental/SetPartitions/test/runtests.jl

```diff
@@ -0,0 +1,6 @@
+using Test
+using Oscar
+
+include("Partition-test.jl")
```

Comment:
I think this `runtests.jl` file can be deleted, as we automatically include all *.jl files inside `test` subdirectories?